src/OpenCV_Mat.jl

```diff
@@ -41,27 +41,35 @@ inline void ptr_val3(cv::Mat &img, int row, int col, std::vector<double> vec)
 
 function pixget(img, row::Int, col::Int)
   (row < 0 || col < 0 || row > rows(img) || col > cols(img)) ? throw(BoundsError()) : nothing
-
-  # Grayscale and binary images
-  val = @cxx ptr_val(img, row, col);
-  println(val)
-  return (val)
-
-  # RGB images
-  vec = @cxx ptr_val3(img, row, col);
-  println([int(at(vec, 0)), int(at(vec, 1)), int(at(vec, 2))])
-  return (vec)
+  image = Images.separate(img);
```

Comment:
The docs at JuliaImages look good. I have no time myself for the Wiki, but thought that it might be something kvmanohar22 could help with if useful for the project...